addons/mail/tools/link_preview.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from lxml import html
+import requests
+
+
+def get_link_preview_from_url(url, request_session=None):
```

Comment:
can you add a docstring to explain the method ? (why `request_session` can be useful, etc)